plugins/modules/cloud/linode/linode.py

```diff
@@ -86,9 +89,17 @@
     description:
     - Set threshold for average IO ops/sec over 2 hour period.
     type: int
+  backupsenabled:
```

Comment:
What's the reason of adding an option that does not do anything to documentation? (aside from removing `undocumented-parameter` from sanity ignores)

Reply:
removing the sanity ignore was the main driver behind that, but as that parameter is to be deprecated, I suppose it does not matter much either way.

Reply:
So there isn't any reason aside from that, but I guess it's not necessary to mention a deprecated option in docs, as this will raises confusion level too ;)
Whoever uses this module will get a deprecated warning thanks to this change, which is enough IMHO.

Reply:
I think it's better to document all options (if they are not hidden for explicitly documented reasons), and explicitly mention in the docs that they are deprecated and will be removed (in $VERSION).